ports/raspberrypi/common-hal/microcontroller/Pin.h

```diff
@@ -34,10 +34,13 @@
 
 #include "peripherals/pins.h"
 
+extern bool cyw_ever_init;
+
```

Comment:
Guard with `CIRCUITPY_CYW43` so it's completely not there if compiling for regular Pi Pico?